webplugin/js/app/kommunicate-ui.js

```diff
@@ -974,8 +974,12 @@ KommunicateUI = {
         }
     },
     activateTypingField: function () {
-        !kommunicateCommons.checkIfDeviceIsHandheld() &&
+        if(kommunicate._globals.inputFocus) {
+            !kommunicateCommons.checkIfDeviceIsHandheld() && null
```

Comment:
what is the purpose of this code?

Reply:
if(kommunicate._globals.inputFocus)

sir, I have added a check for a particular customer such that if he has used a parameter in the Ifarme URL it captures it and prevents the scroll

Reply:
why did you add null there @HrithikKrishna56

Reply:
This is corrected @panwaranuj01